system/system_error_monitor/src/system_error_monitor_core.cpp

```diff
@@ -28,6 +28,34 @@
 
 namespace
 {
+enum class DebugLevel { DEBUG = 0, INFO, WARN, ERROR };
+
+// debug: 0, info: 1, warn: 2, error: 3
+template <DebugLevel debug_level>
+void logThrottledNamed(
+  const std::string & logger_name, const rclcpp::Clock::SharedPtr clock, const double duration,
+  const std::string & message)
+{
+  static std::unordered_map<std::string, rclcpp::Time> last_output_time;
+  if (last_output_time.count(logger_name) != 0) {
+    const auto time_from_last_output = clock->now() - last_output_time.at(logger_name);
+    if (time_from_last_output.seconds() * 1000.0 < duration) {
```

Comment:
Adding the unit would be more readable.

```suggestion
    if (time_from_last_output.seconds() * 1000.0 < duration_ms) {
```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/510/commits/142b5a553a992dc53d0db0231db87439c646f0d9
Done